frontend/packages/knative-plugin/src/components/overview/KnativeServiceResources.tsx

```diff
@@ -1,27 +1,37 @@
 import * as React from 'react';
 import * as _ from 'lodash';
-import { K8sResourceKind, podPhase, PodKind } from '@console/internal/module/k8s';
+import { podPhase } from '@console/internal/module/k8s';
+import { isOverviewTabSection, useExtensions } from '@console/internal/plugins';
+import { AsyncComponent } from '@console/internal/components/utils';
+import { BuildOverview } from '@console/internal/components/overview/build-overview';
 import { PodModel } from '@console/internal/models';
-import { AllPodStatus } from '@console/shared';
+import { AllPodStatus, OverviewItem } from '@console/shared';
 import { PodsOverview } from '@console/internal/components/overview/pods-overview';
 import RevisionsOverviewList from './RevisionsOverviewList';
 import KSRoutesOverviewList from './RoutesOverviewList';
 
 const REVISIONS_AUTOSCALED = 'All Revisions are autoscaled to 0';
 
+const getResourceTabSectionComp = (t) => (props) => (
+  <AsyncComponent {...props} loader={t.properties.loader} />
+);
+
+const usePluginTabSectionResource = (item: OverviewItem) => {
+  const tabSections = useExtensions(isOverviewTabSection);
+  return tabSections
+    .filter((section) => item[section.properties.key])
+    .map((section) => ({
```

Comment:
```suggestion
    .map((section: OverviewTabSection) => ({
```

Reply:
Both the helper function `getResourceTabSectionComp` & the hook `usePluginTabSectionResource`  are a good candidates for having them in a utils file, bcoz it could be reused. Thoughts?